src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -541,4 +565,22 @@ private function checkProductQuantity(
             }
         }
     }
+
+    private function compareItemSubtotal(ProductInterface $product, \Closure $price): void
```

Comment:
I see that very simillar or even the same function is here https://github.com/Sylius/Sylius/pull/12081/files/a2cef239265f11c566a0efbff8514ef2142685d4#diff-b5c163d690bfe47026cba0b16b71bdc7a4e1b0d95352b10522d4b0194d23e85dR555
Maybe we can think about another class that can have same function? 😉

Reply:
This is the same function because this PR is based on #12081